lib/pdk/util/version.rb

```diff
@@ -0,0 +1,34 @@
+require 'pdk/version'
+require 'pdk/cli/exec'
+
+module PDK
+  module Util
+    module Version
+      def self.version_string
+        "#{PDK::VERSION} #{pdk_ref}".strip.freeze
+      end
+
+      def self.pdk_ref
+        ref = pkg_sha || git_ref
```

Comment:
In that case it would be good to know both, so maybe they should just each be conditionally added if available?